src/components/TodoItems.tsx

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import cn from 'classnames';
+import React, {
+  useContext,
+  useState,
+  useEffect,
+  useCallback,
+  useRef,
+} from 'react';
+import { Todo } from '../types/Todo';
+import { TodoContext } from './TodoContext';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItems: React.FC<Props> = React.memo(({ todo }) => {
+  const { setTodos, todos, setChecked } = useContext(TodoContext);
+  const [title, setTitle] = useState(todo.title);
+  const [editing, setEditing] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isEditing, setIsEditing] = useState(false);
```